pom.xml

```diff
@@ -16,7 +16,7 @@
 
 	<properties>
 		<java.version>1.8</java.version>
-		<keycloak.version>9.0.2</keycloak.version>
+		<keycloak.version>11.0.2</keycloak.version>
```

Comment:
11.0.2 results in a crash. Try using the latest available version (13.0.1)

Reply:
Updated!